### PR TITLE
fix: common names map update

### PR DESCRIPTION
**Pull request by anantjain45823**
## Description of the change

> Common Names Map Update 
## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/681)

### CODE REVIEW

File: integrations/Adroll/constants.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ const NAME = "ADROLL";
 const CNameMapping = {
   [NAME]: NAME,
   Adroll: NAME,
-  "Ad roll": NAME,
+  ADROLL: NAME,
```
Comment (MoumitaM) [inline integrations/Adroll/constants.js:5]: Line 5 is not needed as it is already present.

File: integrations/Appcues/constants.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ const NAME = "APPCUES";
 const CNameMapping = {
   [NAME]: NAME,
   Appcues: NAME,
+  APPCUES: NAME,
```
Comment (MoumitaM) [inline integrations/Appcues/constants.js:5]: Not needed.

File: integrations/Braze/constants.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const NAME = "BRAZE";
 const CNameMapping = {
   [NAME]: NAME,
   Braze: NAME,
+  BRAZE: NAME,
```
Comment (MoumitaM) [inline integrations/Braze/constants.js:5]: Not needed. Instead Add "braze".

**Review by MoumitaM — CHANGES_REQUESTED** [review]
Most of the mapping has repetitive values. Remove those values.

**Review by MoumitaM — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Pull request merged**